src/app-layer-parser.c

```diff
@@ -59,12 +59,10 @@ struct AppLayerParserThreadCtx_ {
     void *(*alproto_local_storage)[FLOW_PROTO_MAX];
 };
 
-
 /**
  * \brief App layer protocol parser context.
  */
-typedef struct AppLayerParserProtoCtx_
```

Comment:
looks like this file is reformatted. Please undo that. We don't really want to do it, but if we do it, it should be in a separate commit